cs_util/cfis.py

```diff
@@ -0,0 +1,98 @@
+"""CFIS.
+
+:Name: cfis.py
+
+:Description: This script contains CFIS-specific methods.
+
+:Author: Martin Kilbinger <martin.kilbinger@cea.fr>
+
+"""
+
+
+import re
+import numpy as np
+from astropy import units
+from astropy import coordinates as coords
+
+
+class Cfis(object):
+    """Cfis
+
+    Class for CFIS image properties.
+
+    """
+    size = {'tile' : 0.5 * units.deg}
```

Comment:
Are you planning to expand upon this class? At the moment it looks like overkill for just this size definition.

Reply:
We could merge this with the `image` class from [](https://github.com/CosmoStat/shapepipe/blob/develop/shapepipe/utilities/cfis.py). In this latter library file the size and other quantities are actually defined as global variables, probably also not the best way to do this.